infra/growth/src/resources/rules.js

```diff
@@ -85,9 +90,14 @@ class CampaignRules {
     if (opts.beforeCampaign && opts.duringCampaign) {
       throw new Error('beforeCampaign and duringCampaign args are incompatible')
     }
-    const whereClause = {
-      ethAddress: ethAddress.toLowerCase()
-    }
+
+    // Load any proxy associated with the wallet address.
+    const ownerAddress = ethAddress.toLowerCase()
+    const proxies = await db.Proxy.findAll({ where: { ownerAddress } })
+
+    // Query events from wallet and proxy(ies).
+    const addresses = [ownerAddress, ...proxies.map(proxy => proxy.address)]
```

Comment:
Should we call `toLowerCase()` on proxy addresses as well?

Reply:
The proxy table already stores addresses in lower case so that should not be necessary.